src/server/src/organizations/services/organizations.service.ts

```diff
@@ -35,4 +35,24 @@ export class OrganizationsService extends TypeOrmCrudService<Organization> {
       .getOne();
     return data;
   }
+
+  async getUserOrgsForRegion(
+    regionCode: string,
+    userId: string
+  ): Promise<Organization[] | undefined> {
+    // Returns public data for organization screen
+    const builder = this.repo.createQueryBuilder("organization");
+    const data = await builder
+      .select()
+      .leftJoinAndSelect(
+        "organization.projectTemplates",
+        "projectTemplates",
+        "projectTemplates.isActive = TRUE"
+      )
+      .leftJoinAndSelect("organization.users", "users", "users.id = :userId", { userId: userId })
```

Comment:
We don't want to expose user emails here:
```suggestion
      .leftJoin("organization.users", "users", "users.id = :userId", { userId: userId })
      .addSelect(["users.id", "users.name"])
```